src/main/kotlin/org/sol4k/Constants.kt

```diff
@@ -9,4 +9,12 @@ object Constants {
 
     @JvmStatic
     val ASSOCIATED_TOKEN_PROGRAM_ID = PublicKey("ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL")
+
+    @JvmStatic
+    val COMPUTE_BUDGET__PROGRAM_ID = PublicKey("ComputeBudget111111111111111111111111111111")
+
+    object SysPubkey {
+        @JvmStatic
+        val RENT_PUBKEY = PublicKey("SysvarRent111111111111111111111111111111111")
```

Comment:
Let's move it out to the top level for the `Constants` object and just call `SYSVAR_RENT_ADDRESS` (as it is called in web3.js) to prevent confusion

Reply:
OK